src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -126,12 +126,8 @@ function MoneyRequestParticipantsSelector({
             },
         );
 
-        if (!isPaidGroupPolicy) {
-            return optionList;
-        }
-
         const orderedOptions = OptionsListUtils.orderOptions(optionList, undefined, {
-            sortByReportTypeInSearch: true,
+            sortByReportTypeInSearch: isPaidGroupPolicy,
```

Comment:
I think `config` will be ignored when we're passing `searchValue`  as `undefined` because `orderReportOptions(options: ReportUtils.OptionData[])` doesn't have a `config` parameter